src/server/index.ts

```diff
@@ -1002,21 +1034,24 @@ export class DocumentDriveServer extends BaseDocumentDriveServer {
                 .updateSynchronizationRevisions(
                     drive,
                     syncUnits,
-                    () => this.updateSyncStatus(drive, 'SYNCING'),
+                    () => {
+                        this.updateSyncStatus(drive, 'SYNCING');
+                        this.updateSyncStatus(id, 'SYNCING');
```

Comment:
@acaldas here I'm using the document `id` as the key for the `syncStatus`, but if I remember correctly, I think you said we should use the syncUnit id. Is that correct? let me know so I can push another commit using the syncUnit id instead